CODEOWNERS

```diff
@@ -1 +1 @@
-* @psiemens @Kay-Zee @janezpodhostnik @sideninja
+* @psiemens @Kay-Zee @janezpodhostnik @sideninja @DylanTinianov
```

Comment:
we shouldn't be changing codeowners in code change PRs.

Reply:
Removed